How do you deal with multiple files ?

My app has a class called site in a site.h headder with the site.cpp file and i wish to include that in my main.cpp but if i define the `#define CROW_MAIN` in the site.h i get and error that ```/usr/bin/ld: main.o:(.bss+0x0): multiple definition of `crow::mime_types[abi:cxx11]'; site.o:(.bss+0x0): first defined here
/usr/bin/ld: main.o:(.data+0x0): multiple definition of `crow::detail::dumb_timer_queue::tick'; site.o:(.data+0x0): first defined here``` but if i put the define in only one of them it complains in the other that the define does not exitst. Sorry i posted the issue empty by mistake

well yes but when it compile's `site.cpp` it complains that ```In file included from site.h:2,
                 from site.cpp:1:
dependencies/includes/crow/crow_all.h:39:25: error: declaration of ‘constexpr’ variable ‘crow::VERSION’ is not a definition
   39 |   extern constexpr char VERSION[];``` i can post the full code if that's easier

--- a/include/crow/version.h
+++ b/include/crow/version.h
@@ -3,8 +3,8 @@
 namespace crow {
 
 #ifdef CROW_MAIN
-  constexpr char VERSION[] = "master";
+  char VERSION[] = "master";
 #else
-  extern constexpr char VERSION[];
+  extern char VERSION[];
 #endif
 }

